app/presenters/api/v2/measures/measure_condition_presenter.rb

```diff
@@ -31,7 +31,15 @@ def condition_duty_amount
         end
 
         def duty_expression
-          measure_condition_components.map(&:formatted_duty_expression).join(' ')
+          formatted_components = measure_condition_components.map do |measure_condition_component|
+            if measure_condition_component.small_producers_quotient?
+              measure_condition_component.verbose_duty_expression
+            else
+              measure_condition_component.formatted_duty_expression
+            end
```

Comment:
It seems we should add this logic into a method in the `measure_condition_component` ,

for example

`measure_condition_components.map(&:presented_duty_expression)`

Reply:
Yeah, this is a great suggestion. Implemented this